docs/usage/config-presets.md

```diff
@@ -55,6 +55,7 @@ You can set a Git tag (like a SemVer) to use a specific release of your shared c
 | ------------------------------------------- | -------------------------------- | --------- | ---------------------------- | --------------- | -------------- |
 | GitHub default                              | `github>abc/foo`                 | `default` | `https://github.com/abc/foo` | `default.json`  | Default branch |
```

Comment:
@HonkingGoose Can you open a new docs issue to refactor this file? I think it's very verbose to duplicate every possible preset options for every platform.

I like to have one general table and then below platform specific values if exist.